mmv1/templates/terraform/pre_delete/spanner_instance.go.erb

```diff
@@ -0,0 +1,25 @@
+
+if d.Get("force_destroy").(bool) {
+	// Override GET url to get backups
+	url, err = replaceVars(d, config, "{{SpannerBasePath}}projects/{{project}}/instances/{{name}}/backups")
```

Comment:
Why do we need to override `url` instead of introducing a new variable?

Reply:
I reworked it.